tests/test_busybox.py

```diff
@@ -48,6 +48,47 @@ def test_busybox_image_size(
     )
 
 
+def test_busybox_links(auto_container):
+    """Ensure all binaries in :file:`/bin` are links to :file:`/usr/bin/busybox`."""
+    auto_container.connection.run_expect(
+        [0],
+        'for i in /bin/*; do stat -c "%N" "$i" | grep "/usr/bin/busybox"; done',
+    )
+
+
+def test_busybox(auto_container):
+    busybox = auto_container.connection.run_expect([0], "busybox")
+
+
+def test_true(auto_container):
+    auto_container.connection.run_expect([0], "true")
```

Comment:
How about testing false as well?
```suggestion
def test_true_false_binaries(auto_container):
    auto_container.connection.run_expect([0], "true")
    auto_container.connection.run_expect([1], "false")
```

Reply:
done